images/nginx/rootfs/build.sh

```diff
@@ -600,8 +616,9 @@ writeDirs=( \
   /var/log/nginx \
 );
 
-addgroup -Sg 101 www-data
-adduser -S -D -H -u 101 -h /usr/local/nginx -s /sbin/nologin -G www-data -g www-data www-data
+groupadd -rg 101 www-data
+adduser -u 101 -M -d /usr/local/nginx \
```

Comment:
are the userland tools different? is that why the args are different? if so, please comment upstream commands and write the updated commands separately